app/views/classifications/_classification_featuring.html.erb

```diff
@@ -1,10 +1,15 @@
 <%= content_tag_for(:article, classification_featuring, nil, class: "feature item-#{classification_featuring_counter}") do %>
+
+  <%
+    link_attributes ||= { class: 'govuk-link' }
+    link_attributes[:rel] = "external" if is_external?(classification_featuring.url)
+  %>
   <div class="content">
     <span class="image-holder">
       <%= link_to classification_featuring.image_tag(:s465), classification_featuring.url, title: t("document.read", title: classification_featuring.title), class: 'img'%>
```

Comment:
To avoid two of the same links in a row being read out:

```suggestion
      <%= link_to classification_featuring.image_tag(:s465), classification_featuring.url, title: t("document.read", title: classification_featuring.title), class: "img", aria: { hidden: true } %>
```